crypto/asn1/a_time.c

```diff
@@ -147,10 +147,9 @@ ASN1_GENERALIZEDTIME *ASN1_TIME_to_generalizedtime(ASN1_TIME *t,
     if (!out || !*out) {
         if (!(ret = ASN1_GENERALIZEDTIME_new()))
             goto err;
-        if (out)
-            *out = ret;
-    } else
+    } else {
         ret = *out;
+    }
```

Comment:
Well if you have them on one branch of the "if" then you should have them on both. Technically according to the style guide you should use braces where the body is a compound statement and not otherwise. However I think we have tended to use braces where the body is multi-line (regardless of whether it is compound or not).